app/views/users/_blocked_person.haml

```diff
@@ -0,0 +1,12 @@
+.media.blocked_person{id: person.id}
+  .pull-right
+    = link_to t("users.privacy_settings.stop_ignoring"), block_path(block),  class: "btn btn-danger", method: :delete
+  .media-object.pull-left
+    = person_image_link(person)
```

Comment:
I took this from https://github.com/diaspora/diaspora/blob/develop/app/views/people/_person.html.haml :)